docs/CONTRIBUTING.md

```diff
@@ -271,5 +271,9 @@ kubectl get pvc
 
 TODO - add information about CI after https://github.com/dell/csi-baremetal/issues/562
 
+## New fields
```

Comment:
`New fields` sound confusing to me. Please add something like `Adding new fields to the Kubernetes Custom Resource Definitions`